app/src/main/java/com/example/playlistmaker/media/model/data/db/TrackDbConverter.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.playlistmaker.media.model.data.db
+
+import com.example.playlistmaker.media.model.data.db.entity.TrackEntity
+import com.example.playlistmaker.search.model.domain.Track
+
+class TrackDbConverter {
+    companion object {
```

Comment:
Можно проще сделать:
```suggestion
object TrackDbConverter {
```